### ISSUE TITLE
Finalize option interface

### ISSUE THREAD

**Issue by krlmlr** [tidyverse/duckplyr#264]
Support options in addition to (or instead of) env vars.

**Comment by maelle** [comment]
Actually seeing this, maybe telemetry should not be one file per option as in my PR but instead there could be a duckplyr configuration file? https://blog.r-hub.io/2020/03/12/user-preferences/#using-a-config-file

**Comment by maelle** [comment]
This way, all options would be in one place. Maybe it could somehow be created by default, to show the default options.

**Comment by maelle** [comment]
The setting of options could happen via functions, or manually.

There could be several locations for the configuration file like for an `.Rprofile` (user-level vs project-level).

Having it all in one file means it's easier for users to get an overview but probably also for us to document all options somewhere central?

**Comment by krlmlr** [comment]
Yeah, the user interface vs. the actual storage of the options are two separate questions. I'll think about it and work with what's in the PRs.

**Comment by krlmlr** [comment]
Descoping, the env var interface is good enough.
